editquality/utilities/label_reverted.py

```diff
@@ -69,15 +73,18 @@ def main(argv=None):
     session = mwapi.Session(host, user_agent="ORES revert labeling utility")
 
     verbose = args['--verbose']
-
+    regex = args['--exclude-regex']
     run(revisions, reverteds, session, revert_radius, revert_window, threads,
-        verbose=verbose)
+        regex, verbose=verbose)
 
 
 def run(revisions, reverteds, session, revert_radius, revert_window, threads,
-        verbose=False):
+        regex, verbose=False):
 
     def check_was_damaging_revert(revision):
+        rvprop = ["user"]
+        if regex:
```

Comment:
I could simply add the comment, but I thought it would degrade performance in cases we don't want comment check